src/lib/dateFormatter.ts

```diff
@@ -16,13 +16,13 @@ const monthConverter = {
 	11: 'Dec',
 };
 
-const isLast24Hrs = (date) => {
+const isLast24Hrs = (date: any) => {
```

Comment:
Is there a reason to use `any` instead of `Date` here? Given that we then do a `>=` with it against a `Date` object, does it not need to also be a `Date` object?

Reply:
Good point. I started changing this but it surfaced an issue where this function might not be working so I'm going to raise a ticket for it

Reply:
https://trello.com/c/6aZX3gEr/2322-islast24hours-function-in-discussion

Reply:
I added a fix for this